src/en/proposals/dragonjspider-Wawaspecies.md

```diff
@@ -0,0 +1,64 @@
+# Wawa, or Scugs as a playable roundstart species.
+
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| fairlysadpanda, erisws, cornerdogs, dragonjspider, kiterny, thecatbastard | :information_source: Open PR | TBD |
+
+
+## Overview
+This proposal's plan is to add "Wawa" (Inspired by Scugs) as a playable roundstart species, like lizards or humans. While that is our main goal, if the option is presented instead, a ghost role for them would be nice as well.
+This doc will go into detail about their stats and abilities, and discussing the possible impacts some of these may have in different scenarios.
+Feedback would be greatly appreciated; as there are also some experimental ideas that may be unbalanced, or unfun to play around.
+
+## Background
+We have discussed this a lot on the discord, leading to this proposal (starting from https://discord.com/channels/310555209753690112/1212937078153805844/1220355368228618310). There have been a lot of ideas behind this, and there may be more that were overlooked.
+Feel free to read all of it, but this doc will also summarise what were agreed to be good ideas, as well as the more experimental/system effecting changes. We also understand that there may be some copyright issues with this, so I will attach any permissions to this as well, If and when any are given.
+
+## Copyright permissions
+Currently we have permission from a dev of Downpour, expeditions specifically. Whilst this isn't "proper" approval, more of a semi-approval, we believe it allows us to continue with a project of making a "Scug Inspired" species. The following screenshots are provided courtesy of fairlysadpanda.
+
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/ae7437cf-f4d3-42c1-8798-b8ce24a2725f)
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/69d00309-faae-41e0-a6aa-9e2916052de3)
```

Comment:
Upload these to the repo

Reply:
Sorry uh, new to github... what repo?

Reply:
Oh, nevermind, I never looked into what repo is even short for

Reply:
in ``/src/en/assets/images``

then you can do (for example)
``../assets/images/skug/skug.png``

Reply:
Just to clarify, is there  no need to have the pictures visible in the doc?